### DISCUSSION TITLE
Set custom Font-Family in Snackbar

### DISCUSSION THREAD

**Discussion by maslewandowski** [CommunityToolkit/Maui#1091] [Q&A] [answered]
What is best practice approach to set the font-family of a snack bar?
I registered the custom font in the mauiprogram.cs and now i want to use this font for the snackbar that you provide.

Is that possible and how should i do that?

I've read the documentation and found this:
```
snackbarOptions.Font = Microsoft.Maui.Font.SystemFontOfSize(14);
snackbarOptions.ActionButtonFont = Microsoft.Maui.Font.SystemFontOfSize(14);
```

Thanks!

**Comment by maslewandowski** [comment, marked as answer]
Nevermind: https://github.com/CommunityToolkit/Maui/discussions/1019

**Comment by CliffAgius** [comment]
Closed as answered
